nomad/grapl-core.nomad

```diff
@@ -724,14 +746,16 @@ job "grapl-core" {
       driver = "docker"
 
       config {
-        image        = "${var.container_registry}grapl/node-identifier:${var.node_identifier_tag}"
-        network_mode = "grapl-network"
```

Comment:
grapl-network has to be removed for AWS deploys since the network doesn't exist there